articles/Installation and Upgrading/setup-eslint-on-windows-workstation.md

```diff
@@ -0,0 +1,12 @@
+# Setup Javascript work station
+
+## 1. Prepare PHP Storm
+To install ESLint plugin on PHP Storm go to: File > Settings > Plugins > Browse Repositories and search for ESLint, install and relaunch PHP Storm.
+
+Install Node.js and install ESLint globally with command `npm i -g eslint`.
+
+Configuration file is located at: `package-tao\tao\views\build\.eslintrc.json`
+
```

Comment:
Please use the following convention for paths:
- The root directory is the one that contains all extensions
- Use forward slashes, even when your environment is Windows
The path would then be _tao/views/build/.eslintrc.json_